metabase/metabase
# FE: disable submit button in /admin/databases/<id> if fields are unchanged

Closes #29388 

### Description

Using the `dirty` context property from Formik to compute `FormSubmitButton`’s disabled property.

### How to verify

1. Go to http://localhost:3000/admin/databases/1
2. Submit button should be disabled.
3. Change display name.
4. Submit button should be enabled.
5. Change display name back.
6. Submit button should be disabled.

### Demo

https://github.com/metabase/metabase/assets/116838/8fbef97d-6abe-49a3-9e0a-2b7be69ef998


### Checklist
